### PR TITLE
AST-1202 - responsive test case for button padding

### PR DESCRIPTION
**Pull request by bsfsnehal**
### Description

Added E2E responsive test for button padding
### Screenshots

https://share.getcloudapp.com/kpu6moYe
### How has this been tested?

Execution- npm run test:e2e:interactive — tests/e2e/specs/customizer/global/button/button-padding.test.js
### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/global/button/button-padding.test.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	it( 'button padding should apply correctly', async () => {
+		const buttonPadding = {
+			'theme-button-padding': {
+				desktop: {
+					top: '80',
+					right: '80',
+					bottom: '80',
+					left: '80',
+				},
+				tablet: {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/global/button/button-padding.test.js:14]: value should be different for responsive mode

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/global/button/button-padding.test.js:15]: resolved

**Review by GauravKhupse — APPROVED** [review]

File: tests/e2e/specs/customizer/global/button/button-padding.test.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import { setCustomize } from '../../../../utils/customize';
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setBrowserViewport } from '../../../../utils/set-browser-viewport';
+describe( 'Global button setting under the Customizer', () => {
+	it( 'button padding should apply correctly', async () => {
+		const buttonPadding = {
+			'theme-button-padding': {
```
[the remark this answers is not in the document]
Comment (GauravKhupse) [reply tests/e2e/specs/customizer/global/button/button-padding.test.js:8]: Added commented code for responsive.

[the remark this answers is not in the document]
Comment (bsfsnehal) [reply tests/e2e/specs/customizer/global/button/button-padding.test.js:8]: yes, done this but having issue on github hence commented this code

**Review by SimranChougule1 — APPROVED** [review]

File: tests/e2e/specs/customizer/global/button/button-padding.test.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+				'desktop-unit': 'px',
+				'tablet-unit': 'px',
+				'mobile-unit': 'px',
+			},
+		};
+		await setCustomize( buttonPadding );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( 'form[CLASS*="wp-block-search__"].wp-block-search .wp-block-search__inside-wrapper .wp-block-search__button' );
```
[the remark this answers is not in the document]
Comment (bsfsnehal) [reply tests/e2e/specs/customizer/global/button/button-padding.test.js]: There is no padding set for comment button

**Pull request merged**